lib/validators/movie-list.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).optional(),
+  release_year: Joi.number().min(1878).max(9999).optional()
```

Comment:
You should take heed of @mgartner's [comment](https://github.com/kylekwong/sfmovies/pull/4#discussion_r65566612) about limiting it to integers for the create validator to prevent possible 500s.